googlesearch/__init__.py

```diff
@@ -356,10 +338,9 @@ def search(query, tld='com', lang='en', tbs='0', safe='off', num=10, start=0,
 
         # Prepare the URL for the next request.
         start += num
-        if num == 10:
-            url = url_next_page % vars()
-        else:
-            url = url_next_page_num % vars()
+        url = url_next_page % vars() \
```

Comment:
I think most of these changes are idiomatic and tbh I prefer the old, more readable version, since speed is not an issue here (the bottleneck is clearly the HTTP download)